### PR TITLE
Random SHOWUPDATER thread minute start time

### PR DESCRIPTION
**Pull request by fernandog**
Try to avoid load at 3am (default value)

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by OmgImAlexis** [comment]
@fernandog what's the reason for this?

**Comment by fernandog** [comment]
@OmgImAlexis as I said in the first message: "Try to avoid load at 3am (default value)"

everybody will hit indexer at 03:00am. huge load at 00 minutes

**Comment by medariox** [comment]
I like the idea :+1:

**Comment by OmgImAlexis** [comment]
@fernandog well that explains it. I know it was to avoid the 3am time but I was trying to work out why....

Makes sense then. :+1:

**Comment by labrys** [comment]
I have a helper in mind that will help with this too.

**Comment by fernandog** [comment]
@miigotu rebased

**Comment by labrys** [comment]
@fernandog Consider updating the UI to reflect that it could happen sometime during that hour.  Also consider randomizing the start hour during initial config creation.

File: gui/slick/views/config_general.mako [hunk 1]
```diff
@@ -84,10 +84,11 @@
                         </div>
                         <div class="field-pair">
                             <label for="showupdate_hour">
-                                <span class="component-title">When to update shows</span>
+                                <span class="component-title">choose hour to update shows</span>
```
Comment (medariox) [inline gui/slick/views/config_general.mako]: I would capitalize "Choose"

File: sickbeard/__init__.py [hunk 2]
```diff
@@ -257,7 +258,7 @@
 DEFAULT_DAILYSEARCH_FREQUENCY = 40
 DEFAULT_BACKLOG_FREQUENCY = 21
 DEFAULT_UPDATE_FREQUENCY = 1
-DEFAULT_SHOWUPDATE_HOUR = 3
+DEFAULT_SHOWUPDATE_HOUR = random.randint(1, 5)
```
Comment (medariox) [inline sickbeard/__init__.py]: I would make it 3-5, or 2-5 at max

File: gui/slick/views/config_general.mako (hunk 1 above)
Comment (fernandog) [inline gui/slick/views/config_general.mako]: component-title are not capitalize

Comment (medariox) [inline gui/slick/views/config_general.mako]: image

Comment (fernandog) [inline gui/slick/views/config_general.mako]: no that! the description

**Comment by fernandog** [comment]
@miigotu done

**Pull request merged**